cscs-checks/mch/collectives_halo.py

```diff
@@ -40,6 +40,16 @@ def __init__(self, variant, bench_reference):
                 '-DCUDA_COMPUTE_CAPABILITY="sm_60"'
             ]
             self.build_system.max_concurrency = 8
+        else:
+            self.num_tasks = 4
+            self.num_gpus_per_node = 1
+            self.num_tasks_per_node = 1
+            self.variables['MPICH_RDMA_ENABLED_CUDA'] = '1'
+            self.variables['MV2_USE_CUDA'] = '1'
+            self.build_system.config_opts += [
```

Comment:
Here, don't use the `sm` flag. Since it is generic it does not have to be there.